src/components/code

```diff
@@ -0,0 +1,139 @@
+<!-- // import { link } from 'react-router-dom'
```

Comment:
Please get rid of commented out files. Or at least just don't commit them. They make your code really difficult to navigate as a reviewer!